assets/javascripts/discourse/components/modal/import-pages.js.es6

```diff
@@ -0,0 +1,33 @@
+import Component from "@ember/component";
+import LandingPage from "../../models/landing-page";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import { action } from "@ember/object";
+import bootbox from "bootbox";
+
+export default Component.extend({
```

Comment:
@cabidop This modal throws an error when you try to close it.

Reply:
Good catch, seems that the "cancel" button has a different behaviour than clicking the "X" and it is trying to load a blank page instead of just closing the modal.

Reply:
Fixed 👍